### DISCUSSION TITLE
What does "disabled" do in `UseFormProps`?

### DISCUSSION THREAD

**Discussion by Toshinaki** [react-hook-form/react-hook-form#12067] [Q&A]
As stated in the title, 
https://github.com/react-hook-form/react-hook-form/blob/03f9d7278afce52ecd1fa3c8b2a5e9b8a82b7cd6/src/types/form.ts#L107
we can see a `disabled` props in `UseFormProps`, 
however, I found nothing about it in document
https://react-hook-form.com/ts#UseFormProps

What exactly does this prop do?

I can see that input fields are disabled, if `useForm({disabled: true})`, but not the submit button.

**Comment by jjo10** [comment]
I would like for it to be an official thing and disable all input fields, but I get [this ](https://github.com/orgs/react-hook-form/discussions/10964) symptom when I use it.
